src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -151,13 +151,20 @@ public void getNonBinaryFilesList_directoryWithBinaryFiles_success() {
                 "binaryFileTest/binaryFile.txt", "My Documents/word.docx", "My Documents/pdfDocument.pdf",
                 "My Documents/wordToHtml_files/themedata.thmx", "My Pictures/pngPicture.png");
         GitCheckout.checkoutBranch(config.getRepoRoot(), BRANCH_WITH_BINARY_FILES);
-        Set<Path> files = FileInfoExtractor.getNonBinaryFilesList(config);
+        Set<Path> nonBinaryFiles = FileInfoExtractor.getFilesList(config, false);
 
-        Assert.assertEquals(6, files.size());
+        Assert.assertEquals(6, nonBinaryFiles.size());
         // Non binary files should be captured
-        nonBinaryFilesList.forEach(nonBinFile -> Assert.assertTrue(files.contains(Paths.get(nonBinFile))));
+        nonBinaryFilesList.forEach(nonBinFile -> Assert.assertTrue(nonBinaryFiles.contains(Paths.get(nonBinFile))));
         // Binary files should be ignored
-        binaryFilesList.forEach(binFile -> Assert.assertFalse(files.contains(Paths.get(binFile))));
+        binaryFilesList.forEach(binFile -> Assert.assertFalse(nonBinaryFiles.contains(Paths.get(binFile))));
+
+        Set<Path> binaryFiles = FileInfoExtractor.getFilesList(config, true);
+        Assert.assertEquals(5, binaryFiles.size());
+        // Binary files should be captured
+        binaryFilesList.forEach(binFile -> Assert.assertTrue(binaryFiles.contains(Paths.get(binFile))));
+        // Non binary files should be ignored
+        nonBinaryFilesList.forEach(nonBinFile -> Assert.assertFalse(binaryFiles.contains(Paths.get(nonBinFile))));
```

Comment:
Can we have this test separated? It is not very informative on which part causes the failure should this test fails.

Reply:
Yep didn't realise that. Fixed.